src/main/java/lotto/domain/Lotto.java

```diff
@@ -3,42 +3,39 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
-import java.util.stream.Collectors;
+import java.util.Objects;
 
 public class Lotto implements Iterable<LottoNumber> {
-    private static final int LOTTO_DIGIT = 6;
+    public static final int LOTTO_DIGIT = 6;
     private final List<LottoNumber> lotto;
 
     public Lotto(List<LottoNumber> lotto) {
         validateLottoSize(lotto);
         this.lotto = lotto;
     }
 
+    private void validateLottoSize(List<LottoNumber> lottoNubmers) {
+        if(lottoNubmers.size() != LOTTO_DIGIT) {
+            throw new IllegalArgumentException("로또는 총 " + LOTTO_DIGIT + "개의 번호로 이뤄져 있습니다.");
+        }
+    }
+
     public static Lotto generateLotto(List<Integer> numbers) {
         validateDuplicateLottoNumber(numbers);
         List<LottoNumber> lottoNumbers = new ArrayList<>();
         numbers.forEach(number -> lottoNumbers.add(new LottoNumber(number)));
 
+//        Set<LottoNumber> lottoNumberSet = new TreeSet<>();
+//        numbers.forEach(number -> lottoNumberSet.add(new LottoNumber(number)));
+//
+//        lottoNumbers = lottoNumberSet.stream().collect(Collectors.toList());
```

Comment:
```suggestion
Set<LottoNumber> lottoNumberSet = new TreeSet<>(Comparator.comparingInt(LottoNumber::getLottoNumber));
```
위와 같이 Comparator 를 구현해주세요 !